schema.ts

```diff
@@ -1088,6 +1088,19 @@ export type ContextProperties = Array<PropertyIdentifier<string>>;
 
 export type BasicIndexedProperty = PropertyIdentifier<string>;
 
+/**
+ * A property that will be used to filter the results of a search.
+ * 
+ * For example, if you are trying to query for all emails last modified in the last 30 days,
+ * you would set the filterableProperties to the property that represents the last modified date.
+ * 
+ * Filterable properties must be one of type 
+ * {@link ValueHintType.SelectList} {@link ValueType.Boolean} {@link ValueHintType.DateTime} or 
+ * {@link ValueType.Number} (the number must be an integer)
```

Comment:
Nit, could say that floats will be supported in the near future, but until then, floats will be truncated to ints for filtering.